### PR TITLE
feat: add database debug env var

### PR DESCRIPTION
**Pull request by dni**
makes it visible what db actions happen

screenshot-1711005251

### CODE REVIEW

File: lnbits/app.py [hunk 1]
```diff
@@ -579,6 +579,12 @@ def configure_logger() -> None:
     logging.getLogger("uvicorn.error").handlers = [InterceptHandler()]
     logging.getLogger("uvicorn.error").propagate = False
 
+    logging.getLogger("sqlalchemy").handlers = [InterceptHandler()]
+    logging.getLogger("sqlalchemy.engine.base").handlers = [InterceptHandler()]
+    logging.getLogger("sqlalchemy.engine.base").propagate = False
+    logging.getLogger("sqlalchemy.engine.base.Engine").handlers = [InterceptHandler()]
+    logging.getLogger("sqlalchemy.engine.base.Engine").propagate = False
+
```
Comment (motorina0) [inline lnbits/app.py:587]: why is this required?

Comment (dni) [reply lnbits/app.py:587]: to remove duplications and fit it in better with the other logs.
screenshot-1711005251

Comment (dni) [reply lnbits/app.py:587]: compare to the above in the original screenshot, there were duplications and no formatting

**Review by motorina0 — APPROVED** [review]

**Review by blackcoffeexbt — APPROVED** [review]
